src/smart-components/group/group-table-helpers.js

```diff
@@ -1,27 +1,47 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, useState } from 'react';
 import { Link } from 'react-router-dom';
 import { DateFormat } from '@redhat-cloud-services/frontend-components';
+import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+import { Popover } from '@patternfly/react-core';
+import classNames from 'classnames';
 
-export const createRows = (data, opened, selectedRows = []) => (
-  data.reduce((acc, { uuid, name, roleCount, principalCount, modified, platform_default: isPlatformDefault }) => ([
-    ...acc,
-    {
-      uuid,
-      isPlatformDefault,
-      cells: [
-        <Fragment key={ uuid }>
-          <Link to={ `/groups/detail/${uuid}` }>
-            { name }
-          </Link>
-        </Fragment>,
-        roleCount,
-        principalCount,
-        <Fragment key={ `${uuid}-modified` }>
-          <DateFormat date={ modified } type="relative" />
-        </Fragment>
-      ],
-      selected: Boolean(selectedRows && selectedRows.find(row => row.uuid === uuid))
-    }
-  ]), [])
-);
-
+export const createRows = (data, opened, selectedRows = []) => {
+  const [ isPopoverVisible, setPopoverVisible ] = useState(false);
+  return (
+    data.reduce((acc, { uuid, name, roleCount, principalCount, modified, platform_default: isPlatformDefault }) => ([
+      ...acc,
+      {
+        uuid,
+        isPlatformDefault,
+        cells: [
+          <Fragment key={ uuid }>
+            <Link to={ `/groups/detail/${uuid}` }>
+              { name }
+            </Link>
+            { isPlatformDefault &&
+            <Popover
+              zIndex="110"
+              position="right"
+              className="pf-c-default-group-popover"
+              isVisible={ isPopoverVisible }
+              shouldClose={ () => setPopoverVisible(false) }
+              hideOnOutsideClick={ true }
+              bodyContent={ 'This group contains the roles that all users in your organization inherit by default.' }
```

Comment:
No need to wrap it in object.
```suggestion
              bodyContent="This group contains the roles that all users in your organization inherit by default."
```